.gitignore

```diff
@@ -2,6 +2,8 @@ _site
 *.swp
 .DS_Store
 Gemfile.lock
-
+.jekyll-metadata
+.tool-versions
+semver.org.iml
```

Comment:
Please drop them, files/dirs specific to your env should go to your global gitignore config.

Reply:
Fixed and cleaned up. 
Sorry for the inconvenience :(